run.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use PHPOxford\SpiresIrc\Irc\Connection;
+use PHPOxford\SpiresIrc\Irc\Message;
+use PHPOxford\SpiresIrc\Irc\User;
+use PHPOxford\SpiresIrc\IrcClient;
+use PHPOxford\SpiresIrc\Plugins\Greetings;
+use PHPOxford\SpiresIrc\Plugins\PingPong;
+use PHPOxford\SpiresIrc\Plugins\Time;
+use PHPOxford\SpiresIrc\Plugins\Welcome;
+
+require_once 'vendor/autoload.php';
+
+error_reporting(E_ALL);
+
+/* Allow the script to hang around waiting for connections. */
+set_time_limit(0);
+
+/* Turn on implicit output flushing so we see what we're getting as it comes in. */
+ob_implicit_flush();
+
+$client = new IrcClient(
+    new Connection(
+        '#phpoxford',
+//        '##martinsbottesting',
+        'irc.freenode.com'
+    ),
+    new User(
+        'spires',
+        'spires',
+        'Spires ALPHA'
+    )
+);
+
+$client->connect();
+
+
+// Ping Pong
+$client->addPlugin(new PingPong());
```

Comment:
Possibly not something for this PR but should our bot use some sort of autoloading pattern for plugins?